### PR TITLE
CART-89 doc: Update doc for DER_HLC_SYNC

### PR DESCRIPTION
**Pull request by frostedcmos**
- Update documentation for DER_HLC_SYNC

Skip-test: true

Signed-off-by: Alexander Oganezov <alexander.a.oganezov@intel.com>

### CODE REVIEW

File: doc/admin/troubleshooting.md [hunk 1]
```diff
@@ -238,6 +240,11 @@ is the same between each component.  Depending on your situation, you will need
 to either update the DAOS Agent or the libdaos.so to the newer version in order
 to maintain compatibility with each other.
 
+When DER_HLC_SYNC is received, it means that the sender and receiver clocks
+are out of sync with each other for more than 1 second.  In order to correct this
```
Comment (jolivier23) [inline doc/admin/troubleshooting.md:244]: is 1 second a fixed amount or does it depend on the system?

**Review by jolivier23 — APPROVED** [review]

File: doc/admin/troubleshooting.md (hunk 1 above)
Comment (frostedcmos) [reply doc/admin/troubleshooting.md]: updated wording

**Review by jolivier23 — APPROVED** [review]

**Comment by johannlombardi** [comment]
@liw please review. Thanks in advance.

File: doc/admin/troubleshooting.md [hunk 2]
```diff
@@ -32,6 +32,8 @@ errors are documented in the table below.
 |DER_NOTDIR|1027|Not a directory
 |DER_EVICTED|1032|Rank has been evicted
 |DER_DOS|1034|Denial of service
+|DER_BAD_TARGET|1035|Incorrect target for the RPC
+|DER_HLC_SYNC|1037|HLC Synchronization error
```
Comment (liw) [inline doc/admin/troubleshooting.md:36]: s/Synchronization/synchronization/?

File: doc/admin/troubleshooting.md [hunk 3]
```diff
@@ -238,6 +240,14 @@ is the same between each component.  Depending on your situation, you will need
 to either update the DAOS Agent or the libdaos.so to the newer version in order
 to maintain compatibility with each other.
 
+When DER_HLC_SYNC is received, it means that sender and receiver HLC timestamps
+are off by more than epsilon value (1 second by default).
```
Comment (liw) [inline doc/admin/troubleshooting.md:244]: s/epsilon/maximum system clock offset/, as epsilon is an internal term?

File: doc/admin/troubleshooting.md [hunk 4]
```diff
@@ -238,6 +240,14 @@ is the same between each component.  Depending on your situation, you will need
...
+Maximum epsilon offset can be changed using crt_hlc_epsilon_set(uint64_t epsilon)
+API.
```
Comment (liw) [inline doc/admin/troubleshooting.md:246]: This is not for "admin". I think we can remove these two lines.

File: doc/admin/troubleshooting.md [hunk 5]
```diff
@@ -238,6 +240,14 @@ is the same between each component.  Depending on your situation, you will need
...
+
+In order to correct this situation run OS-supplied clock synchronization service
+like NTP or manually correct clocks.
```
Comment (liw) [inline doc/admin/troubleshooting.md:249]: "All servers' clocks must be synchronized to the same reference time, using services like NTP."

"Manually" is not going to work in the long run.

**Review by jolivier23 — APPROVED** [review]

**Review by liw — APPROVED** [review]
Thanks, Alex.

**Pull request merged**